cpp/src/glm/preprocess.cuh

```diff
@@ -42,20 +43,20 @@ void preProcessData(const raft::handle_t &handle, math_t *input, int n_rows,
          "Parameter n_rows: number of rows cannot be less than two");
 
   if (fit_intercept) {
-    Stats::mean(mu_input, input, n_cols, n_rows, false, false, stream);
-    Stats::meanCenter(input, input, mu_input, n_cols, n_rows, false, true,
-                      stream);
+    raft::stats::mean(mu_input, input, n_cols, n_rows, false, false, stream);
+    MLCommon::Stats::meanCenter(input, input, mu_input, n_cols, n_rows, false,
+                                true, stream);
 
-    Stats::mean(mu_labels, labels, 1, n_rows, false, false, stream);
-    Stats::meanCenter(labels, labels, mu_labels, 1, n_rows, false, true,
-                      stream);
+    raft::stats::mean(mu_labels, labels, 1, n_rows, false, false, stream);
+    MLCommon::Stats::meanCenter(labels, labels, mu_labels, 1, n_rows, false,
```

Comment:
This split (mean in raft, meanCenter in mlcommon) really makes me want to move the rest of stats ASAP!